src/Standards/Generic/Tests/PHP/DisallowShortOpenTagUnitTest.php

```diff
@@ -24,14 +24,9 @@ class DisallowShortOpenTagUnitTest extends AbstractSniffUnitTest
      */
     protected function getTestFiles($testFileBase)
     {
-        $testFiles = [$testFileBase.'1.inc'];
-
-        $option = (boolean) ini_get('short_open_tag');
-        if ($option === true || defined('HHVM_VERSION') === true) {
```

Comment:
The condition and the `if`/`else` logic needs to stay, just the ` || defined('HHVM_VERSION') === true` part needs to be removed.

Reply:
Oops, should have caught that. Fixed now, thanks.